### ISSUE TITLE
HYBRID mode

### ISSUE THREAD

**Issue by normanr** [benb116/Etch#1]
Did you experiment with a HYBRID mode where the MANUAL mode is active while AUTO mode controls the knobs (but the direct drawing from the pattern is disabled)? It might be interesting to see how those drawings come out.

**Comment by benb116** [comment]
I'm not sure I understand. Are you suggesting that I make the motors turn the knobs automatically to follow an existing pattern, then force the drawing on screen to be "sourced" purely from the movement of the knobs instead of drawn directly from the point file? In theory that could be done, but I suspect the results would be of lower quality. The motors are not driven with high precision and there is also a bit of error in sensing their position.

**Comment by normanr** [comment]
Yes, that's correct. I expect the loss of precision, but I thought it might be fun to see how bad it actually looks. You could even add an option to randomly (eg: 10% of drawings) use this mode instead of the high precision mode.
